oracle/oci-utils
# init script to auto start kvm network at boot

When user creates creates a Libvirt network we also create an initd script 
to make the network available after reboot.

test done :
1 - create a libvrit network
 * check the network is up 
 * check that init script is in place
 * check that after reboot network is also up

test in progress :
 create a network , create a guest then test connectivity before and after a reboot
